sample/src/main/kotlin/com/slack/circuit/sample/petdetail/PetDetail.kt

```diff
@@ -137,18 +142,36 @@ private fun RenderImpl(state: PetDetailScreen.State) {
       }
       PetDetailScreen.State.NoAnimal -> TODO()
       is PetDetailScreen.State.Success -> {
-        LazyColumn(modifier = Modifier.padding(padding)) {
-          item {
-            CircuitContent(
-              PetPhotoCarousel(
-                name = state.name,
-                photoUrls = state.photoUrls,
-                photoUrlMemoryCacheKey = state.photoUrlMemoryCacheKey,
-              )
+        if (isLandscape) {
+          Row(
+            horizontalArrangement = Arrangement.SpaceEvenly,
+          ) {
+          CircuitContent(
+            PetPhotoCarousel(
+              name = state.name,
+              photoUrls = state.photoUrls,
+              photoUrlMemoryCacheKey = state.photoUrlMemoryCacheKey,
             )
+          )
+            LazyColumn {
+              item { Text(text = state.name, style = MaterialTheme.typography.displayLarge) }
+              item { Text(text = state.description) }
+            }
```

Comment:
Another nit: we should extract these to a shared function

```kotlin
@Composable
fun ColumnScope.PetDetails(state) {

}
```

Then it can be reuse in both cases